runtime/nls/j9vm/j9vm.nls

```diff
@@ -2448,3 +2448,33 @@ J9NLS_VM_JAVA_COMPILER_WARNING_XINT.explanation=The java.compiler system propert
 J9NLS_VM_JAVA_COMPILER_WARNING_XINT.system_action=Setting the java.compiler system property is ignored.
 J9NLS_VM_JAVA_COMPILER_WARNING_XINT.user_response=Use the -Xint option to disable the JIT.
 # END NON-TRANSLATABLE
+
+# Message for an IllegalAccessError when the sub class can't access its super class or interface
+# arguments 1 and 2 are the sub class name
+# arguments 3 is superclass or superinterface
+# arguments 4 and 5 are the super class name
+# arguments 6 is " " for Java 11+, or empty for Java 8
+# arguments 7 is the module name of the sub class or empty for Java 8
+# arguments 8 is " of" for Java 11+, or empty for Java 8
+# arguments 9 and 10 are the classloader name of the sub class
+# argument 11 is the module name of the super class or empty for Java 8
+# arguments 12 and 13 are the classloader name of the super class
+J9NLS_VM_CLASS_LOADING_ERROR_INVISIBLE_SUPERCLASS_OR_INTERFACE=class %2$.*1$s cannot access its %3$s %5$.*4$s (%2$.*1$s is in%6$s%7$s%8$s loader %10$.*9$s; %5$.*4$s is in%6$s%11$s%8$s loader %13$.*12$s)
```

Comment:
Please reorganize the message to avoid repeating the class and superclass names.

Reply:
This message format is similar with RI `IllegalAccessError`, do you have any recommendation?

Reply:
The problem is that `class a.se.java_lang_exception.ExtendsDefaultVisibilityInterface` is misleading: the message is not referring to a class, but an interface.

Reply:
Use `interface` for the `interface` classes, and `class` for others, this matches `Class.toString()`
https://github.com/eclipse-openj9/openj9/blob/f255919f1f639eb4b64238b12e5fcd5e03320565/jcl/src/java.base/share/classes/java/lang/Class.java#L2947